hack/e2e/run-rp-and-e2e.sh

```diff
@@ -211,9 +211,31 @@ delete_e2e_cluster() {
         ./cluster delete
     else
         go run ./hack/cluster delete
+        go run ./hack/cluster deleteApp
     fi
 }
 
+get_cluster_sp() {
+    echo "########## Downloading SP secrets ##########"
+
+    az keyvault secret download --vault-name=aro-e2e-principals \
```

Comment:
Not blocking the PR, but we should add a TODO here for eventually grabbing a single SP among the entire pool